packages/st2/Makefile

```diff
@@ -33,14 +33,10 @@ else
 	PYTHON_ALT_BINARY := python
 endif
 
-# Don't include Mistral runner on Bionic or EL8
-ifeq ($(DEB_DISTRO),bionic)
-	RUNNERS := $(shell ls -I mistral* ../contrib/runners)
-else ifeq ($(EL_VERSION),8)
+# Don't include Mistral runner
+# TODO: Go back to all runners when mistral removed from st2
```

Comment:
As I understand we'll need another PR in st2-packages, once the mistral removal happens in the st2 repository.

Reply:
Yes - it's a bit chicken and egg.  It will still work when we take out mistral from contrib/runners in ST2 - but we should tidy up afterwards.
But this code can be used before and after the ST2 changes.